source/includes/common/scanner.rst

```diff
@@ -0,0 +1,43 @@
+.. start-scanner-speed-config
+
+You can adjust how MinIO balances the scanner performance with read/write operations using either the :envvar:`MINIO_SCANNER_SPEED` environment variable or the :mc-conf:`scanner speed <scanner.speed>` configuration setting.
+
+.. end-scanner-speed-config
+
+
+.. start-scanner-speed-values
+
+MinIO utilizes the scanner for :ref:`bucket replication <minio-bucket-replication>`, :ref:`site replication <minio-site-replication-overview>`, and :ref:`lifecycle management <minio-lifecycle-management>` tasks.
+
+Valid values include:
+
+.. list-table::
+   :stub-columns: 1
+   :widths: 30 70
+   :width: 100%
+   
+   * - ``fastest``
+     - Removes scanner wait on read/write latency, allowing the scanner to operate at full-speed and IOPS consumption.
+       This setting may result in reduced read/write performance for IOPS-limited systems.
```

Comment:
Nit - thinking about it we can just mirror `fast` here and call it done w.r.t. the IOPS bit.